apis/apps/v1alpha1/nebulacluster_common.go

```diff
@@ -434,3 +435,30 @@ func generateConfigMap(c NebulaClusterComponentter) *corev1.ConfigMap {
 
 	return cm
 }
+
+func mergeStringMaps(overwrite bool, ms ...map[string]string) map[string]string {
```

Comment:
This is a utility function, move it to pkg/util/maputil more better, what do you think?

Reply:
In the future, it may come out separately to create a golang mod or repo, so consider making him less dependent.
Or we will provide a golang pkg in the future that does not depend on Nebula Operator?
Or as you said, move it to `pkg/util/maputil` ?
What do you think?

Reply:
We need make a plan to refactor this part, you can commit another pr to complete it.